app/models/storage.rb

```diff
@@ -298,8 +308,8 @@ def self.max_qitems_per_scan_request
     ::Settings.storage.max_qitems_per_scan_request
   end
 
-  def self.max_parallel_storage_scans_per_host
-    ::Settings.storage.max_parallel_scans_per_host
+  def self.max_parallel_storage_scans_per_ems
+    ::Settings.storage.max_parallel_scans_per_ems
```

Comment:
You should probably just kill this class method as it's a useless abstraction layer...Not needed for this PR.